docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -278,10 +279,10 @@ the ESPW.
 ## Deployment into a Kubernetes cluster
 
 These commands administer a deployment of the central components ---
-the kcp server, PKI, and the central KubeStellar components --- in a
-Kubernetes cluster that will be referred to as "the hosting cluster".
-These commands are framed as "kubectl plugins" and thus need to be
-explicitly or implicitly given a kubeconfig file for the hosting
+the space provider server, PKI, and the central KubeStellar components 
+--- in a Kubernetes cluster that will be referred to as "the hosting 
+cluster".  These commands are framed as "kubectl plugins" and thus need 
```

Comment:
A "kubectl plugin" is something that is given _one_ Kubernetes client configuration via all the usual command line flags and environment variable (`KUBECONFIG`) for kubectl. For the KubeStellar commands that need _one_ kube client config we _chose_ to make them kubectl plugins. Anything that becomes or remains a kubectl plugin should be something that is given _one_ kube client config and that giving is via the usual command line flags and envar.